tests/unit/test_paper_canary_strategy.py

```
@@ -0,0 +1,120 @@
+from pms.actuator.adapters.paper import PaperActuator
+from pms.config import PMSSettings
+from pms.controller.factory import ControllerPipelineFactory
+from pms.core.enums import RunMode
+from pms.core.models import MarketSignal, Portfolio
+from pms.strategies.paper_canary import PAPER_CANARY_STRATEGY_ID, build_paper_canary_strategy
+
+
+def _signal(*, yes_price: float = 0.50, best_ask: float = 0.52) -> MarketSignal:
+    return MarketSignal(
+        market_id="canary-market-8",
+        token_id="canary-token-8",
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "token_id"

(S106)